Source/WebCore/platform/SharedBuffer.h

```diff
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include "SharedMemory.h"
```

Comment:
Would a forward declaration suffice? You included this in both the header and the cpp.

Reply:
just a forward declaration wasn't working. I removed it from the cpp but left it in the header